routes/view.js

```diff
@@ -88,11 +88,12 @@ exports.testView = {
       payload(server, request, done) {
         const email = request.params.email;
         const testPath = `${server.settings.app.views.path}/${email}/test.json`;
+        const disableTracking = (request.query.disableTracking !== 'false');
```

Comment:
It kind of is. However - enableMetrics is global and can't be easily over written ... and this would allow disabling the tracking on a specific view.